{
    "cve": {
        "data_type": "CVE",
        "data_format": "MITRE",
        "data_version": "4.0",
        "CVE_data_meta": {
            "ID": "CVE-2020-10082",
            "ASSIGNER": "cve@mitre.org"
        },
        "problemtype": {
            "problemtype_data": [
                {
                    "description": []
                }
            ]
        },
        "references": {
            "reference_data": [
                {
                    "url": "https:\/\/about.gitlab.com\/releases\/2020\/03\/04\/gitlab-12-dot-8-dot-2-released\/",
                    "name": "https:\/\/about.gitlab.com\/releases\/2020\/03\/04\/gitlab-12-dot-8-dot-2-released\/",
                    "refsource": "MISC",
                    "tags": []
                },
                {
                    "url": "https:\/\/about.gitlab.com\/releases\/2020\/03\/04\/gitlab-12-dot-8-dot-2-released\/index.html",
                    "name": "https:\/\/about.gitlab.com\/releases\/2020\/03\/04\/gitlab-12-dot-8-dot-2-released\/index.html",
                    "refsource": "CONFIRM",
                    "tags": []
                }
            ]
        },
        "description": {
            "description_data": [
                {
                    "lang": "en",
                    "value": "GitLab 12.2 through 12.8.1 allows Denial of Service. A denial of service vulnerability impacting the designs for public issues was discovered."
                }
            ]
        }
    },
    "configurations": {
        "CVE_data_version": "4.0",
        "nodes": []
    },
    "impact": [],
    "publishedDate": "2020-03-13T17:15Z",
    "lastModifiedDate": "2020-03-13T17:22Z"
}